medusa/tv/series.py

```diff
@@ -1569,26 +1569,26 @@ def load_imdb_info(self):
         self.imdb_info['countries'] = self.imdb_info.get('countries', '')
         self.imdb_info['country_codes'] = self.imdb_info.get('country_codes', '')
 
-        # If the show has no year, IMDb returned something we don't want
-        if not imdb_obj or not imdb_obj.year:
-            log.debug(u'{id}: IMDb returned none or invalid info for {imdb_id}, skipping update.',
+        imdb_obj = imdb_api.get_title(self.imdb_id)
```

Comment:
Does it still makes sense to call it `imdb_obj`?

Reply:
Well, it is coming from IMDb and it is obviously an object so I don't see why not. Would imdb_dict be better? imdb_info? I can't really find a good name for it, but I'm open to suggestions.

Reply:
Yeah, I was thinking something like `imdb_info`